daml-assistant/src/DAML/Assistant/Cache.hs

```diff
@@ -24,13 +26,29 @@ newtype CacheKey = CacheKey String
 newtype CacheTimeout = CacheTimeout NominalDiffTime
     deriving (Show, Eq, Ord, Y.FromJSON)
 
+data UpdateCheck
+    = UpdateCheckNever
+    | UpdateCheckEvery CacheTimeout
```

Comment:
Can we not just use System.Time.Extra.Seconds as the type here? I find NominalDiffTime a horrible type to work with...

Reply:
:+1: